faiss/gpu/CMakeLists.txt

```diff
@@ -285,13 +285,13 @@ if(FAISS_ENABLE_RAFT)
   set_source_files_properties(
     GpuDistance.cu
     StandardGpuResources.cpp
-    impl/RaftFlatIndex.cu
-    impl/RaftIVFFlat.cu
-    impl/RaftIVFPQ.cu
-    utils/RaftUtils.cu
+    impl/CuvsFlatIndex.cu
+    impl/CuvsIVFFlat.cu
+    impl/CuvsIVFPQ.cu
+    utils/CuvsUtils.cu
     TARGET_DIRECTORY faiss
     PROPERTIES COMPILE_OPTIONS "-fvisibility=hidden")
```

Comment:
Is this still necessary with cuvs?

Reply:
Yes that is still needed to avoid any errors due to duplicate CUDA symbols.